src/commands/Suggestion Commands/AcceptCommand.ts

```diff
@@ -64,6 +64,16 @@ export const run: RunFunction = async (client, message, args) => {
 			color: 'GREEN',
 		})
 	);
+	
+	const author: User = client.users.cache.get(Suggestion.User);
+	author.send(
```

Comment:
We need to write a check to see if the author is in the cache. A simple `if(author)` would work.

Reply:
Updated to add check.